test/Nancy.Tests/Unit/Bootstrapper/NancyBootstrapperBaseFixture.cs

```diff
@@ -327,7 +327,7 @@ protected override INancyEngine GetEngineInternal()
         {
             if (this.ShouldThrowWhenGettingEngine)
             {
-                throw new ApplicationException("Something when wrong when trying to compose the engine.");
+                throw new Exception("Something when wrong when trying to compose the engine.");

```

Comment:
Na, fair enough. Let's use `Exception` since it's only in the context of the test itself